sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -928,7 +928,7 @@ def is_fast_reboot_enabled():
 
     if "system" in keys:
         output = subprocess.check_output(['sonic-db-cli', 'STATE_DB', 'get', "FAST_REBOOT|system"], universal_newlines=True)
-        if "1" in output:
+        if "enable" in output:
```

Comment:
@arfeigin can you point me to the PR where this change is made in STATE_DB?

Reply:
Sure, mentioned in the description: https://github.com/sonic-net/sonic-utilities/pull/2621, this one and the sonic-buildimage PR should be merged first.